src/Symfony/Component/DependencyInjection/Loader/PhpFileLoader/ServicesConfigurator.php

```diff
@@ -0,0 +1,103 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+
+use Symfony\Component\DependencyInjection\Alias;
+use Symfony\Component\DependencyInjection\ChildDefinition;
+use Symfony\Component\DependencyInjection\ContainerBuilder;
+use Symfony\Component\DependencyInjection\Definition;
+use Symfony\Component\DependencyInjection\Loader\PhpFileLoader;
+use Symfony\Component\DependencyInjection\Reference;
+
+/**
+ * @author Nicolas Grekas <p@tchwork.com>
+ *
+ * @method InstanceofConfigurator instanceof($fqcn)
+ */
+class ServicesConfigurator extends AbstractConfigurator
+{
+    const FACTORY = 'services';
+
+    private $defaults;
+    private $instanceof;
+    private $container;
+    private $loader;
+
+    public function __construct(ContainerBuilder $container, PhpFileLoader $loader, array &$instanceof)
+    {
+        $this->defaults = new Definition();
+        $this->container = $container;
+        $this->loader = $loader;
+        $this->instanceof = &$instanceof;
+        $instanceof = array();
+    }
+
+    /**
+     * @return DefaultsConfigurator
+     */
+    public function defaults()
+    {
+        return new DefaultsConfigurator($this, $this->defaults = new Definition());
+    }
+
+    /**
+     * @return InstanceofConfigurator
+     */
+    protected function setInstanceof($fqcn)
+    {
+        $this->instanceof[$fqcn] = $definition = new ChildDefinition('');
+
+        return new InstanceofConfigurator($this, $definition, $fqcn);
+    }
+
+    /**
+     * @return ServiceConfigurator|ServiceRootConfigurator|AliasConfigurator
```

Comment:
That's what I did at first, but it means having "addAlias" be proposed all the time.
Since aliases are not that common, I preferred doing it this way - also because `->add('id', ref('target')` makes it clear to me what the direction of the alias is (vs `addAlias('id', 'target')`, which personally always stops me a few seconds to remember in which order id and target are)